mkdocs.yml

```diff
@@ -73,89 +62,98 @@ nav:
+              - Use GraphQL over HTTP: Pantheon-API/GraphQL.md
+              - Authenticate JSON-RPC Requests: Pantheon-API/Authentication.md
+            - Consensus Protocols:
+              - Use Clique: Consensus-Protocols/Clique.md
+              - Use IBFT 2.0: Consensus-Protocols/IBFT.md
+              - Connect to Quorum IBFT 1.0 Network: Consensus-Protocols/QuorumIBFT.md
+            - Privacy API Methods:
+              - Use EEA-compliant Privacy: Privacy/How-To/EEA-Compliant.md
+              - Use Pantheon-extended Privacy: Privacy/How-To/Pantheon-Privacy.md
+              - Create and Manage Privacy Groups: Privacy/How-To/Create-Manage-Privacy-Groups.md
+              - Access Private and Privacy Marker Transactions: Privacy/How-To/Access-Private-Transactions.md
+            - Permissioning:
+              - Use Local Permissioning: Permissions/Local-Permissioning.md
+              - Update Onchain Permissioning Whitelists: Permissions/Onchain-Permissioning/Updating-Whitelists.md
+            - Client Libraries:
+              - Use the web3.js-eea Client Library: Privacy/How-To/eeajs.md
+            - Filters:
+              - Access Logs Using JSON-RPC API: Using-Pantheon/Accessing-Logs-Using-JSON-RPC.md
+        - Configure Pantheon:
+           - Specify Options in a Configuration File: Configuring-Pantheon/Using-Configuration-File.md
+           - Configure a Free Gas Network: Configuring-Pantheon/FreeGas.md
+           - Predeploy a Contract in the Genesis File: Configuring-Pantheon/Contracts-in-Genesis.md
+        - Send Transactions:
+           - Use Wallets for Key Management: Using-Pantheon/Account-Management.md
+           - Create and Send Transactions: Using-Pantheon/Transactions/Transactions.md
+           - Create and Send Private Transactions: Privacy/How-To/Creating-Sending-Private-Transactions.md
+           - Include Revert Reason in Transaction Receipts: Using-Pantheon/Transactions/Revert-Reason.md
+        - Deploy for Production:
+          - Use Ansible to Deploy Pantheon: Deploying-Pantheon/Ansible.md
+          - Use Prometheus for Monitoring: Monitoring/Monitoring-Performance.md
+          - Use Alethio Lite Explorer: Monitoring/Alethio/Lite-Block-Explorer.md
+          - Use Alethio EthStats Lite Network Monitor: Monitoring/Alethio/Lite-Network-Monitor.md
+          - Configure High Availability of APIs: Deploying-Pantheon/High-Availability.md
+          - Deploy Permissioning Management Dapp: Permissions/Onchain-Permissioning/Production.md
+        - Find and Connect to Peers:
+          - Start a Bootnode: Configuring-Pantheon/Networking/Bootnodes.md
+          - Configure Ports for Access: Configuring-Pantheon/Networking/Configuring-Ports.md
+          - Manage Peers: Configuring-Pantheon/Networking/Managing-Peers.md
+          - Use UPnP: Configuring-Pantheon/Networking/Using-UPnP.md
+        - Develop on Pantheon:
+          - Pass JVM Options: Configuring-Pantheon/Passing-JVM-Options.md
```

Comment:
I don't think this is development related.  Either configuration or deploy for production.  Although the article itself should probably be more focussed on things like changing memory allocation and other common Java options.  Attaching a debugger is usually just done within the IDE.

Reply:
If we update the article as suggested (focussed on things like changing memory allocation and other common Java options) then does it still belong under Configuration?  Or are those things relevant to developing on Pantheon?

Reply:
Under configuration.